sys/riscv/riscv/pmap.c

```diff
@@ -2013,6 +2013,18 @@ pmap_remove_kernel_l2(pmap_t pmap, pt_entry_t *l2, vm_offset_t va)
 	    __func__, l2, oldl2));
 }
 
+static inline void
+pmap_page_dirty(pt_entry_t tpte, vm_page_t m)
```

Comment:
Is `tpte` a MIPSism? The RISC-V pmap calls it `entry` whenever it's not a specific level PTE.

Reply:
It probably is a MIPSism.  It's unfortunately not quite true that `tpte` isn't in `sys/riscv/riscv/pmap.c` (blame tells me to xref 9d5cba36c among others that all look to be markj's fault... so at least I have some company in my "RISC-V is just a different MIPS, right?" naming).  In any case, I've eliminated this use of `tpte`. :)